### PR TITLE
Add exit codes to wp

### PR DESCRIPTION
**Pull request by fortunac**
Adds exit codes to WP:
unsat -> 0
sat -> 1
unknown -> 2

### CODE REVIEW

File: wp/plugin/wp.ml [hunk 1]
```diff
@@ -431,8 +431,10 @@ let callback
     })
   in
   Parameters.validate params files >>= fun params ->
-  Analysis.run params files ctxt >>= fun () ->
-  Ok ()
+  Analysis.run params files ctxt >>= function
+  | Z3.Solver.UNSATISFIABLE -> exit 0
```
Comment (jtpaasch) [inline wp/plugin/wp.ml:435]: Hmm...I don't know if BAP needs to do any special cleanup after a custom command returns. If so, maybe it's not a good idea to force a premature exit here.

I wonder if there's a way to return `Error ...` with specific exit codes. Maybe a question for Ivan?

Or, maybe we could return `Ok ()` for UNSAT, `Error ...` for SAT, and then `exit 2` for UNKNOWN?

**Review by jtpaasch — APPROVED** [review]
Marking this approved, but there's a comment/question. Do with it what you will!

File: wp/plugin/wp.ml (hunk 1 above)
Comment (fortunac) [reply wp/plugin/wp.ml:435]: That's a very good point. I'll take a look into it!

**Comment by jtpaasch** [comment]
Fantastic work on fixing this.

**Pull request merged**